src/extension/webView.ts

```diff
@@ -76,22 +76,17 @@ async function getNpmVersion(): Promise<string> {
     exec('npm --version', (error, stdout) => {
       if (error) {
         reject(error);
-        return;
       } else {
-        resolve(stdout);
+        const npmVersion = stdout.match(/\d+\.\d+\.\d+/g);
+        npmVersion ? resolve(npmVersion[0]) : reject(error);
```

Comment:
1. Why use an expression? Here we can be explicit and use a statement.
2. `error` is obviously undefined. I think in the reject we should put in an adequate error; `reject(new Error('Could not find a version string in the output'))`